### DISCUSSION TITLE
columnSpan not working for Chart Widget in Dashboard

### DISCUSSION THREAD

**Discussion by michele-grifa** [filamentphp/filament#11611] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.39

### How can we help you?

I'm trying to show two chart widget side by side in the dashboard. They have bot a columnSpan of 6 but they show always full size. The same happen if I use a chart and a stat widget.

Someone have replicated this behaviour?

**Comment by leandrocfe** [comment, marked as answer]
https://filamentphp.com/docs/3.x/panels/dashboard#customizing-the-widgets-grid
If you are using the default columns (2) on the dashboard, just use columnSpan =1

**Reply by michele-grifa** [reply]
Thanks, I was missing the facts that the default columns were 2.
